.env.ci.example

```diff
@@ -22,3 +22,8 @@ SENDGRID_API_KEY=123
 
 # (optional) override devise hours after which link expires (defaults to 6)
 HOURS_PASSWORD_RESET_LINK_EXPIRES_AFTER=8
+
+# Heroku only! Set to turn off devDependencies pruning
+# See https://devcenter.heroku.com/articles/troubleshooting-node-deploys#missing-modules
+# for more info
+YARN_PRODUCTION=false
```

Comment:
Re `.env`, @exbinary is going to work on refactoring those so all possible/default vars are in .env (and then we don't need the ci file anyway, right?). It's my fault for conflating w heroku here.

Reply:
Yeah that makes sense! We'll need this line regardless of the FA issue, though; it's to prevent heroku from pruning dev dependencies that are needed to compile the client bundles

Reply:
This is done!